### ISSUE TITLE
Lucas.

### ISSUE THREAD

**Issue by joshrio** [sceene/sprint-planning#195]
Hey Lucas, 

I am traveling (sending this from the car) so I won’t be online until tomorrow morning. We need to finish all issues this week as we’re going on holiday next Monday. The critical issues are the high priority, and low priority will be changed and prioritized tomorrow morning. 

Here’s the answers to your questions, let me know if there’s anything else missing. 

1. Theres an empty menu cell on the Menu Page
Can you give me the steps to reproduce this please?

Click on “more” from a menu card or menu from the navigation on a profile page.

2. Add the Rating in even if it's not available from the feed.
What should be added if it's not available?’

Rating is available on the profile page. Load it into the ratings on the profile rating stat. 

3. I spent almost 2 hours on this: Actions Sheets have various bugs. It's caused by my implementation of the infinite horizontal scrolling over a third party code. I'm moving on, and I'll get back to this later on. I'm thinking maybe taking care of this on the plane on wednesday

This is the least important thing. Leave it too last.

4. @joshrio can you explain me what you mean by Flicker is still happening.?

See below.

![flicker mov](https://cloud.githubusercontent.com/assets/12651511/11802449/9af3d486-a2a5-11e5-8c0c-9a635a1a7041.gif)

**Comment by LucasCoelho** [comment]
1. _Theres an empty menu cell on the Menu Page Can you give me the steps to reproduce this please?_
Cannot reproduce with the given steps

**Comment by LucasCoelho** [comment]
_Add the Rating in even if it's not available from the feed. What should be added if it's not available?’
Rating is available on the profile page. Load it into the ratings on the profile rating stat._
Steve will be working on adding rating to the feed soon. It's nothing but a waste of time (twice, adding and removing when it will be working) to work on this. Consider it "pending on API"

**Comment by joshrio** [comment]
I cant actually show you because the profile pages all crash. Please fix this and upload a new version.

**Comment by joshrio** [comment]
@LucasCoelho 

Nop problem with waiting for Steve on Ratings. I've added responses to the other questions here https://github.com/sceene/sprint-planning/issues/189 so i'm closing this issue.
